lib/handlers/check_channel_connection_handler.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:async';
+
+import 'package:breez/bloc/account/account_bloc.dart';
+import 'package:breez/logger.dart';
+import 'package:breez/services/device.dart';
+import 'package:breez/services/injector.dart';
+import 'package:breez/theme_data.dart' as theme;
+import 'package:breez/widgets/flushbar.dart';
+import 'package:flushbar/flushbar.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class CheckChannelConnection {
+  static final _instance = CheckChannelConnection._internal();
+
+  factory CheckChannelConnection() => _instance;
+
+  CheckChannelConnection._internal();
+
+  Timer _notReadyTimer;
+  Flushbar _flushbar;
+  StreamSubscription<bool> _subscription;
+
+  void startListen(
+    BuildContext context,
+    AccountBloc accountBloc,
+  ) {
+    ServiceInjector().device.eventStream.distinct().listen((event) {
+      switch (event) {
+        case NotificationType.RESUME:
```

Comment:
I did run to background and backing, and it behaves correctly, but to be honest this kind of situation need more test from different devices IMHO because it can be biased to my device specific configuration.

Reply:
If I can Help anyhow just let me know how.